tests/test_optics.py

```diff
@@ -643,7 +643,7 @@ def test_thin_lens():
 	focal_length = 300e-1
 	lens = ThinLens(focal_length, lambda x : 1.5, 1e-6)
 	aperture = evaluate_supersampled( circular_aperture(5e-2), grid, 8 )
-	assert abs( (lens.focal_length - focal_length)/focal_length - 1.0 ) < 1e-10
+	assert abs((lens.focal_length - focal_length)/focal_length) < 1e-10
```

Comment:
How did this pass the test before?

Reply:
No clue? Are the tests in this file included in the unit tests?

Reply:
I think so. Anything in the `/tests` directory is auto-enumerated, as long as the function starts with `test_`. Let me check if it ran on the old CI.

Reply:
Oh, it didn't pass. The CI tests failed for the commit before this one, with an error in this function. So all is fine.

Reply:
I found it. In commit #67b6ab9 I added more tests, and that commit did not pass the unit tests. So a commit later I fixed the bugs.